packages/uniforms-bootstrap3/__tests__/ListDelField.tsx

```diff
@@ -46,3 +46,11 @@ test('<ListDelField> - correctly reacts on click', () => {
   expect(wrapper.find('i').simulate('click')).toBeTruthy();
   expect(onChange).toHaveBeenLastCalledWith('x', ['x', 'z']);
 });
+
+test('<ListDelField> - correctly reacts on keyboard enter key', () => {
+  const element = <ListDelField name="x.1" />;
+  const wrapper = mount(element, context());
+
+  wrapper.find('[role="button"]').props().onKeyDown?.({ key: 'Enter' } as any);
+  expect(onChange).toHaveBeenLastCalledWith('x', ['x', 'z']);
```

Comment:
Before I go ahead and add tests for all the files I wanted to check a few things with you.

1. `simulate` is deprecated and not recommended by the author. It will also be removed in `enzyme@4` if it ever gets released. More info [here](https://github.com/enzymejs/enzyme/issues/2173). This is why I used `props().onKeyDown` directly.
2. The `onKeyDown` has a very weird type for it's argument which expects a full (and hacked) version of `SyntheticEvent` with all the keys but works just fine with a simple object. This is why I used casting to `any` here.
3. Now that the component is more semantically verbose I also changed the selector to be a little more descriptive and less error-prone to changes. I could also change the other tests to use it as it is pretty much no effort change.

@radekmie @Monteth

Reply:
1. I'd rather stick to what we do use now and then migrate all tests at once.
2. OK; shouldn't be the case with 1.
3. Sure, go ahead!